crates/common/consensus/lean/src/state.rs

```diff
@@ -304,6 +328,8 @@ impl LeanState {
                     "Skipping attestations by Validator {}",
                     attestation.validator_id,
                 );
+                inc_int_counter_vec(&ATTESTATIONS_INVALID_TOTAL, &[]);
+                stop_timer(validation_timer);
```

Comment:
we don't need to call stop_timer here https://docs.rs/prometheus/0.13.4/prometheus/struct.HistogramTimer.html ``This timer can be stopped and observed at most once, either **automatically (when it goes out of scope)** or manually``

Same for the other few places below